pom.xml

```diff
@@ -40,10 +40,10 @@
 
     <licenses>
         <license>
-            <name>Apache License, Version 2.0</name>
+            <name>Apache-2.0</name>
+            <comments>Apache License, Version 2.0</comments>
```

Comment:
Why is that? (Note that some people _do_ rely on specific license identifiers in POMs for assessing compliance.)

Reply:
Not sure why it was changed now and in this PR, but I do notice that the POM reference does say
> Using an SPDX identifier as the license **name** is recommended.
https://maven.apache.org/pom.html#Licenses

Reply:
Fair enough, I did look at the SPDX list yesterday, but didn't notice `Apache-2.0` is the identifier. Funny how the example in the _Licenses_ section of the POM reference shows the usual way of referring to the Apache 2.0 license (which we used to use as well), which doesn't comply with their own recommendation :-)

Reply:
agreed. I think it might be me who lifted the original snippet and then told by someone else recently what we have is not highly recommended.